commit 1fcd5e01d26e99461d4f00b660d43632a81a3306
Author: Tony Lindgren <tony@atomide.com>
Date:   Wed Aug 8 01:07:03 2018 -0700

    ARM: OMAP2+: Fix null hwmod for ti-sysc debug
    
    [ Upstream commit 4769c003e0fcff0ee001a9102e2605bdaa5880f0 ]
    
    We may call omap_hwmod_parse_module_range() with no hwmod allocated yet
    and may have debug enabled. Let's fix this by checking for hwmod before
    trying to use it's name.
    
    Fixes: 6c72b3550672 ("ARM: OMAP2+: Parse module IO range from dts for legacy
    Signed-off-by: Tony Lindgren <tony@atomide.com>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/arch/arm/mach-omap2/omap_hwmod.c b/arch/arm/mach-omap2/omap_hwmod.c
index 2ceffd85dd3d..7f759abcf49c 100644
--- a/arch/arm/mach-omap2/omap_hwmod.c
+++ b/arch/arm/mach-omap2/omap_hwmod.c
@@ -2220,7 +2220,7 @@ int omap_hwmod_parse_module_range(struct omap_hwmod *oh,
 	size = be32_to_cpup(ranges);
 
 	pr_debug("omap_hwmod: %s %s at 0x%llx size 0x%llx\n",
-		 oh->name, np->name, base, size);
+		 oh ? oh->name : "", np->name, base, size);
 
 	res->start = base;
 	res->end = base + size - 1;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1257_1fcd5e01d26e99461d4f00b660d43632a81a3306.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1257_1fcd5e01d26e99461d4f00b660d43632a81a3306.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
